core/variant/variant_utility.cpp

```diff
@@ -749,6 +749,14 @@ struct VariantUtilityFunctions {
 		return String(error_names[error]);
 	}
 
+	static inline String typeof_string(Variant::Type type) {
+		if (type < 0 || type >= Variant::VARIANT_MAX) {
+			return String("(invalid type)");
```

Comment:
I wonder if it would be better to simply return an empty String in this error situation? Would make error checking a bit easier in cases where you might get an invalid type from somewhere.

Reply:
I agree with this, but because of the similarity with the [`error_string()`](https://docs.godotengine.org/en/stable/classes/class_@globalscope.html#class-globalscope-method-error-string) function, I wanted to keep consistency as it returns `"(invalid error code)"` string on failure.

Reply:
Also it can be only used by giving it one `int` parameter and the only way of receiving a bad `Enum` value is by the user directly passing in `int` values, instead of receiving them from [`typeof()`](https://docs.godotengine.org/en/stable/classes/class_@globalscope.html#class-globalscope-method-typeof), so that is why I think the [`error_string()`](https://docs.godotengine.org/en/stable/classes/class_@globalscope.html#class-globalscope-method-error-string) works like that.